### PR TITLE
Updated PR template with new embed standards

### LINKED ISSUE #99: Update pull request template with new media embedding standards

### PR DESCRIPTION
**Pull request by Conor12345**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -6,18 +6,19 @@ Thank you for your submission!
 Name: <!--(to be dispayed in blog)-->
 
 
-##### Requierments for media in submissions
+#### Requirements for media in submissions:
 <!-- Remove items that do not apply. For completed items, change [ ] to [x]. -->
 
 - [ ] Images are hosted on Imgur <!--(recommended width:896px - height:504px)-->
-  - [ ] Use provided Imgur embed `{% include imgur.html id="WsUV4DK" %}` where `id` is last part of Imgur URL https://imgur.com/gallery/`WsUV4DK`
+  - [ ] Use provided Imgur embed `![](https://i.imgur.com/flSlSwB.png)`
```
Comment (DrMagPie) [inline .github/PULL_REQUEST_TEMPLATE.md]: describing where id is coming from might help some people

**Review by DrMagPie — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (Conor12345) [reply .github/PULL_REQUEST_TEMPLATE.md]: Ye could do, not sure exactly how to describe it tho...

**Comment by Conor12345** [comment]
@DrMagPie Where does this `` link come from? I can find the normal imgur one but not where the i.imgur link comes from.

**Comment by DrMagPie** [comment]
It depends. At times it is last part of url and some times you can get it wow inspector

**Pull request merged**